### PR TITLE
Q1 2024: as good a time as any for a little polishing

### PR DESCRIPTION
**Pull request by mckern**
Bringing dependencies up to date and preparing to review some of these PRs.

### CODE REVIEW

**Comment by bhavanki** [comment]
*2024 😃

File: README.md [hunk 1]
```diff
@@ -47,27 +46,27 @@ view of a file with very long lines. More example pairs can be found in the
 
 ## Version support
 
-The latest version of `golines` requires golang 1.18 or newer due to generics-related dependencies.
+Newer releases of `golines` require golang 1.21 or newer due to generics-related dependencies.
```
Comment (bhavanki) [inline README.md:49]: Very minor nit: The need for 1.21 isn't specifically for generics-related dependencies like 1.18 or higher is.

**Review by bhavanki — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (mckern) [reply README.md:49]: I didn't feel like rewriting a chunk of the README last night. Let's see if that changes today.

Comment (mckern) [reply README.md]: README updated

**Review by jiaxin-ge — APPROVED** [review]

**Review by asaf-erlich — APPROVED** [review]

**Pull request merged**